ci/checks/style.sh

```diff
@@ -38,16 +38,31 @@ source ci/common/prep-init-env.sh ${WORKSPACE}
 logger "Run Copyright header check..."
 ./ci/checks/check_copyright.py
 
+# Python
 logger "Run Python/Cython formatting check..."
 python -m pip install -r ./ci/checks/python-style-requirements.txt
 source pyclaragenomics/style_check
 
+logger "Run Python documentation generation..."
+mkdir -p pyclaragenomics/docs; cd pyclaragenomics/docs
+sphinx-quickstart -p "Clara Genomics Analysis SDK" -v "x.y.z" -a "NVIDIA Corportation" -q --ext-autodoc --sep
+##### Update sphinx conf.py with path for modules
+echo "import os" >> temp.py
```

Comment:
Maybe we should use `sed` in order to append commands to the beginning of the `conf.py` file instead of creating a temp file?  What do you think?
https://stackoverflow.com/a/9533736

Reply:
Neat! Updated